client/ayon_core/plugins/publish/extract_review.py

```diff
@@ -598,7 +665,10 @@ def prepare_temp_data(self, instance, repre, output_def):
             "input_allow_bg": input_allow_bg,
             "with_audio": with_audio,
             "without_handles": without_handles,
-            "handles_are_set": handles_are_set
+            "handles_are_set": handles_are_set,
+            "ext": ext,
+            "explicit_frames": [],  # absolute paths to rendered files
```

Comment:
What does the key `"explicit_frames"` hold? Comment says it's absolute paths, but key is `"explicit_frames"`?